app/code/Magento/ExtendedOrder/composer.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
We should explicitly mark this package as abandoned, as we aren't going to support it once 2.3.2 is released.
Something like `"abandoned": true` would be sufficient.